python/tk_framework_adobe_utils/win_32_api.py

```diff
@@ -12,6 +12,7 @@
 A minimal set of win32 functions to manage toolkit dialogs under Windows.
 """
 import ctypes
+from ctype import wintypes
```

Comment:
Seems we have to do this inorder to be able to do `cytypes.wintypes`, importing `ctypes` alone is not enough.

Reply:
🤷‍♂